src/ParticleDA.jl

```diff
@@ -361,7 +368,7 @@ function init_filter(filter_params::FilterParameters, model_data, nprt_per_rank:
     domain_size = get_grid_domain_size(model_data)
     cell_size = get_grid_cell_size(model_data)
 
-    grid = Grid(size...,cell_size...,domain_size...)
+    grid = Grid(size[1],size[2],cell_size[1],cell_size[2],domain_size[1],domain_size[2])
```

Comment:
This is assuming again that `size`, `cell_size` and `domain_size` have specific dimensions, isn't it?

Reply:
If we try to pass three dimensions to init_offline_matrices() we still get an error. Therefore, I just pass the two horizontal components